### PR TITLE
Debug captures on heroku

### PR DESCRIPTION
**Pull request by leppert**

### CODE REVIEW

File: services/heroku/heroku_settings.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
 AWS_QUERYSTRING_AUTH = False
 
 # archive creation
+PHANTOMJS_BINARY = '/app/vendor/phantomjs/bin/phantomjs' # celery won't find this otherwise
```
Comment (leppert) [inline services/heroku/heroku_settings.py]: @jcushman I'm not thrilled about hardcoding this, but the celery worker doesn't seem to have phantomjs in its path for some reason ([see rollbar](https://rollbar.com/perma-staging/perma-staging/items/10/)) despite the fact that Django seems to find it just fine from the shell. Any ideas?

Comment (jcushman) [inline services/heroku/heroku_settings.py]: It looks like the PATH is relative: 

```
~ $ echo $PATH
/app/.heroku/python/bin:/usr/local/bin:/usr/bin:/bin:vendor/phantomjs/bin
```

So it probably works depending on current working directory. 

Relative PATH is an anti-pattern, right? Presumably this is being set by the phantomjs buildpack we're using. Maybe fork it and send the change upstream? Or just switch us to another solution if you think that makes sense. Thanks!

(On the horizon are (a) upgrading to PhantomJS 2.0 or (b) just switching to headless Firefox, which I'm leaning toward. So the simplest fix is probably best for now.)

**Comment by leppert** [comment]
@jcushman this is fixed and ready to go in

**Comment by jcushman** [comment]
Thanks!

**Pull request merged**